lib/drivers/medtronic600/medtronic600Simulator.js

```diff
@@ -482,15 +488,15 @@ class Medtronic600Simulator {
             restoredBasal.deviceTime =
               sundial.formatDeviceTime(sundial.parseFromFormat(activeSuspend.deviceTime).getTime() +
               activeSuspend.duration);
-            if (restoredBasal.duration < 0) {
-              throw new Error('Should never get a duration of less than zero');
-            } else if (restoredBasal.duration > 0) {
+            Medtronic600Simulator.checkDuration(restoredBasal);
+            if (restoredBasal.duration > 0) {
```

Comment:
What happens if duration is === 0? Is it just dropped? Should we log this to console or throw an error?